build/mvn

```diff
@@ -22,7 +22,7 @@ _DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Preserve the calling directory
 _CALLING_DIR="$(pwd)"
 # Options used during compilation
-_COMPILE_JVM_OPTS="-Xms2g -Xmx2g -Xss8m -XX:ReservedCodeCacheSize=512m"
+_COMPILE_JVM_OPTS="-Xms2g -Xmx2g -Xss8m -XX:ReservedCodeCacheSize=1g -Xss128m"
```

Comment:
Do we meet stack overflow? The default stack size is 1m, 128m is too large in my view

Reply:
No evidence for s.o. 128 is from spark

Reply:
but I suspect that the flink tests cause this as the builds mostly end up with killed there

Reply:
I guess it's caused by heap OOM, let wait for CI results